tests/Joli/JoliCi/JobTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Joli\JoliCi;
+
+class JobTest extends \PHPUnit_Framework_TestCase
+{
+    public function testConstructInitialisesAllTheFields()
+    {
+        $projectMock     = uniqid();
+        $strategyMock    = uniqid();
+        $uniqMock        = uniqid();
+        $parametersMock  = array('test' => uniqid());
+        $descriptionMock = 'test';
+        $createdMock     = new \DateTime();
+        $servicesMock    = array('service' => uniqid());
+
+        $jobMock = new Job(
+            $projectMock,
+            $strategyMock,
+            $uniqMock,
+            $parametersMock,
+            $descriptionMock,
+            $createdMock,
+            $servicesMock
+        );
+
+        $this->assertAttributeSame($projectMock, 'project', $jobMock);
+        $this->assertSame($strategyMock, $jobMock->getStrategy());
+        $this->assertSame($uniqMock, $jobMock->getUniq());
+        $this->assertSame($parametersMock, $jobMock->getParameters());
+        $this->assertSame($descriptionMock, $jobMock->getDescription());
+        $this->assertSame($createdMock, $jobMock->getCreated());
+        $this->assertSame($servicesMock, $jobMock->getServices());
+    }
+
+    public function testAddAndGetServices()
+    {
+        $jobMock =
+            $this->getMockBuilder('Joli\JoliCI\Job')
+                 ->disableOriginalConstructor()
+                 ->setMethods(null)
+                 ->getMock();
```

Comment:
How about using a real `Job` instance here instead of mocking the SUT? (see also http://stackoverflow.com/questions/14648451/mocking-the-sut-itself)

Perhaps adding a dependency on the previous case can help avoiding code duplication?